src/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -281,7 +281,13 @@ public void StartStopWithTimestamp()
         [Fact]
         public void StopWithoutTimestamp()
         {
-            var startTime = DateTime.UtcNow.AddSeconds(-1);
+            // DateTime.UtcNow is not precise on some platforms, but Activity stop time is precise
+            // in this test we set start time, but not stop time and check duration.
+            //
+            // Let's set start time to be 1 sec + possible DateTime.UtcNow error and check that duration is at least 1 sec.
+            // There is another test (ActivityDateTimeTests.StartStopReturnsPreciseDuration) 
+            // that checks duration precision on netfx.
+            var startTime = DateTime.UtcNow.AddMilliseconds(-1020);
```

Comment:
If you have the time I would prefer that we define a constant MaxClockErrorMSec = 20 and us it.   But also structure the use of this so that we only use this in the actual time comparisions (thus stick the original experiment (use 1000 msec), but change the assert below to be   Duration.TotalMSec >= 1000 - MaxClockErrorMSec.  

This keeps the logic 'clean, and (when MaxClockError == 0 then things look simple)